pandas/tests/io/formats/test_to_latex.py

```diff
@@ -365,7 +365,7 @@ def test_to_latex_longtable(self, frame):
 \midrule
 \endhead
 \midrule
-\multicolumn{3}{r}{{Continued on next page}} \\
```

Comment:
can you make sure we are testing for 0, 1, 2, 3 columns (you don't have to fully compare the output, just assert that the multicolumn directive is correct)

Reply:
Just added test cases for 1 and 3 columns. We already had a test case with 2 columns. I don't think tables with 0 columns make any sense. What do you think?

Reply:
sure they do, an empty dataframe is possible

Reply:
@jreback : The code has a deliberate short-circuit for an empty `DataFrame`, and this pattern will not be followed for 0 columns in fact.   I added a couple of tests for that short-circuit.